components/Feature/SupportSummary/index.js

```diff
@@ -222,15 +223,66 @@ const SupportSummary = ({
                 residentInfo={residentInfo}
                 formType="summary"
               />
-              <TextArea
-                value={emailBody}
-                label="Add a note for the resident"
-                name="support-summary-note"
-                rows="20"
-                onChange={value => {
-                  setEmailBody(value);
-                }}
-              />
+              <div className="govuk-form-group">
+                <fieldset className="govuk-fieldset">
+                  <div className="govuk-!-padding-top-4">
+                    <label>
+                      <strong>How would you like to share information with the resident?</strong>
+                    </label>
+                  </div>
+                  <div className="govuk-radios">
+                    <div className="govuk-radios__item">
+                      <input
+                        className="govuk-radios__input"
+                        id="summary-sharing-method-email"
+                        name="summary-sharing-method"
+                        type="radio"
+                        value="email"
+                        onClick={() => {
+                          setEmailBody(updateEmailBody());
+                        }}
+                        required
+                      />
+                      <label
+                        className="govuk-label govuk-radios__label"
+                        for="summary-sharing-method-email">
+                        Email
+                      </label>
+                    </div>
+                    <div className="govuk-radios__item">
+                      <input
+                        className="govuk-radios__input"
+                        id="summary-sharing-method-letter"
+                        name="summary-sharing-method"
+                        type="radio"
+                        value="letter"
+                        onClick={() => {
+                          setEmailBody(
+                            updateEmailBody(undefined, undefined, undefined, undefined, 'letter')
+                          );
+                        }}
+                        required
+                      />
+                      <label
+                        className="govuk-label govuk-radios__label"
+                        for="summary-sharing-method-letter">
+                        Letter
+                      </label>
+                    </div>
+                  </div>
+                </fieldset>
+              </div>
+              <div className="govuk-!-padding-top-4">
+                <TextArea
+                  value={emailBody}
+                  label="Add a note for the resident"
+                  name="support-summary-note"
+                  rows="20"
+                  onChange={value => {
+                    setEmailBody(value);
+                  }}
+                />
+              </div>
```

Comment:
This is the same as the lines that appear to be removed in this file (225-233) except wrapped in a div for formatting